_posts/2021-03-00-image-segmentation.md

```diff
@@ -0,0 +1,345 @@
+---
+layout: post
+title: Image segmentation with Dask
+author: Genevieve Buckley
+tags: [imaging]
+theme: twitter
+---
+{% include JB/setup %}
+
+## Executive Summary
+
+A basic image segmentation pipeline
+
+[dask-image](http://image.dask.org/en/latest/)
+
```

Comment:
Do these informative posts need a call to action? (Or was that more for the community building ones)